<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSettingsTextMessageInlineComponent</class>
 <widget class="QFrame" name="CSettingsTextMessageInlineComponent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>293</width>
    <height>46</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Overlay message settings</string>
  </property>
  <layout class="QGridLayout" name="gl_OverlayMessages">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="lbl_Overlay">
     <property name="toolTip">
      <string>Overlay</string>
     </property>
     <property name="text">
      <string>Overlay:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QCheckBox" name="cb_Supervisor">
     <property name="toolTip">
      <string>supervisor</string>
     </property>
     <property name="text">
      <string>supervisor</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QCheckBox" name="cb_Private">
     <property name="text">
      <string>private</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QCheckBox" name="cb_Selcal">
     <property name="text">
      <string>SELCAL</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QCheckBox" name="cb_Frequency">
     <property name="toolTip">
      <string>frequency</string>
     </property>
     <property name="text">
      <string>frequency</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QCheckBox" name="cb_Focus">
     <property name="text">
      <string>focus</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QCheckBox" name="cb_All">
     <property name="toolTip">
      <string>all messages as overlay</string>
     </property>
     <property name="text">
      <string>all</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2" alignment="Qt::AlignLeft">
    <widget class="QWidget" name="wi_Buttons" native="true">
     <layout class="QHBoxLayout" name="hl_Buttons">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="pb_Disable">
        <property name="toolTip">
         <string>disable all overlay message</string>
        </property>
        <property name="text">
         <string>disable</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_Reset">
        <property name="text">
         <string>reset</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cb_Private</tabstop>
  <tabstop>cb_Supervisor</tabstop>
  <tabstop>cb_Selcal</tabstop>
  <tabstop>cb_Frequency</tabstop>
  <tabstop>pb_Disable</tabstop>
  <tabstop>pb_Reset</tabstop>
  <tabstop>cb_All</tabstop>
  <tabstop>cb_Focus</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
